release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -398,11 +398,30 @@ In {product-title} 4.8, the `rhcos4-moderate` profile is now complete. The `ocp4
 
 The {product-title} Ingress Controller is upgraded to HAProxy version 2.2.13.
 
+[discrete]
 [id="ocp-4-8-coreDNS-version-update"]
 ==== CoreDNS update to version 1.8.1
 
 In {product-title} 4.8, CoreDNS uses version 1.8.1, which has several bug fixes, renamed metrics, and dual-stack IPv6 enablement.
 
+[discrete]
+[id="ocp-4-8-etcd-zap-logger"]
+==== etcd now uses the zap logger
+
+In {product-title} 4.8, etcd now uses zap as the default logger instead of capnslog. Zap is a structured logger that provides machine consumable JSON log messages. You can use `jq` to easily parse these log messages.
+
+If you have a log consumer that is expecting the capnslog format, you might need to adjust it for the zap logger format.
+
+.Example capnslog format ({product-title} 4.7)
```

Comment:
Should the examples have `[source,terminal]` since they are code blocks?

Reply:
You know, I did debate that, but didn't end up using it because it's a log message, and not necessarily in the terminal. But I'm looking at a few other places that show log messages and most do use it. So I'll go ahead and update to use it :)